Fixed minor issues in Pong tutorial book (#1051)
agbrs/agb: book/src/pong/09_keeping_score.md

Firstly, let's add the score to the `Paddle` objects:

```rust
pub struct Paddle {
    pos: Vector2D<Fixed>,
    health: i32,
}
```

and in the `new()` function, initialise it to 3.

> We are gonna use it for displaying health as hearts and decrementing it when ball touches either left side or right side of screen. Add these functions to `Paddle` struct.

Reviewer: ```suggestion
We're going to use it for displaying the health as hearts and decrementing it when ball touches the side of the screen. Add these functions to `Paddle` struct.
```

@gwilymk might have a better suggestion.

Author: Thanks! That sounds much better.

```rust
pub fn health(&self) -> i32 {
    self.health
}

pub fn decrement_health(&mut self) {

>     self.health -= 1

Reviewer: ```suggestion
    self.health -= 1;
```

```rust
struct Paddle {
    pos: Vector2D<Fixed>,

>     health: i32,

Reviewer 2: do you think we should add a `get_health(&self)` method to Paddle instead of making this `pub`? (legit question, I'm not sure what the best answer here is :grin:)

Reviewer 2: I think `get_health()` and `decrement_health()` is much nicer. Makes it clear what we expect you to do with the health field. Good suggestion!

```rust
Object::new(sprites::BALL.sprite(0))
    .set_pos(self.pos.round())

>     .set_priority(Priority::P1)

Reviewer 2: We've used it as `Priority` in the above cases, so it feels like this should've been imported already?
